src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+

```

Comment:
Avoid starting the interface implementation with an empty line. Remove this redundant empty line as per the checklist guidelines.